recipe/windows.patch

```diff
@@ -1,39 +1,40 @@
---- magick/studio.h
-+++ magick/studio.h
-@@ -471,10 +471,10 @@
- */
- 
- #if defined(HasLTDL)
--#  define SupportMagickModules
-+//#  define SupportMagickModules
- #elif !defined(__MINGW32__) && !defined(__MINGW64__)
- #  if defined(MSWINDOWS) && defined(_DLL)
--#    define SupportMagickModules
-+//#    define SupportMagickModules
- #  endif
- #endif
- 
-
---- magick/static.h
-+++ magick/static.h
-@@ -158,7 +158,7 @@
-   RegisterXPMImage(void),
-   RegisterXVImage(void),
- #if defined(_VISUALC_)
--  RegisterXTRNImage(void),
-+  //RegisterXTRNImage(void),
- #endif
-   RegisterXWDImage(void),
-   RegisterYUVImage(void),
-
---- magick/static.c
-+++ magick/static.c
-@@ -188,7 +188,7 @@
-   STATICM("XCF",RegisterXCFImage,UnregisterXCFImage),
-   STATICM("XPM",RegisterXPMImage,UnregisterXPMImage),
- #if defined(_VISUALC_)
--  STATICM("XTRN",RegisterXTRNImage,UnregisterXTRNImage),
-+  //STATICM("XTRN",RegisterXTRNImage,UnregisterXTRNImage),
- #endif /* defined(_VISUALC_) */
- #if defined(HasX11)
-   STATICM("XWD",RegisterXWDImage,UnregisterXWDImage),
+diff -Naur GraphicsMagick-1.3.35.orig/magick/static.c GraphicsMagick-1.3.35/magick/static.c
+--- GraphicsMagick-1.3.35.orig/magick/static.c	2020-02-23 12:39:56.000000000 -0300
++++ GraphicsMagick-1.3.35/magick/static.c	2020-11-13 12:14:30.950044681 -0300
+@@ -188,7 +188,7 @@
+   STATICM("XCF",RegisterXCFImage,UnregisterXCFImage),
+   STATICM("XPM",RegisterXPMImage,UnregisterXPMImage),
+ #if defined(_VISUALC_)
+-  STATICM("XTRN",RegisterXTRNImage,UnregisterXTRNImage),
++  // STATICM("XTRN",RegisterXTRNImage,UnregisterXTRNImage),
+ #endif /* defined(_VISUALC_) */
+ #if defined(HasX11)
+   STATICM("XWD",RegisterXWDImage,UnregisterXWDImage),
+diff -Naur GraphicsMagick-1.3.35.orig/magick/static.h GraphicsMagick-1.3.35/magick/static.h
+--- GraphicsMagick-1.3.35.orig/magick/static.h	2020-02-23 12:39:56.000000000 -0300
++++ GraphicsMagick-1.3.35/magick/static.h	2020-11-13 12:14:59.534421766 -0300
+@@ -158,7 +158,7 @@
+   RegisterXPMImage(void),
+   RegisterXVImage(void),
+ #if defined(_VISUALC_)
+-  RegisterXTRNImage(void),
++  // RegisterXTRNImage(void),
```

Comment:
@seanyen I don't know the first thing about Windows, so I appreciate if you can take a quick look at this patch. BTW, there is one more reference to `RegisterXTRNImage(void)` down below, should I comment that one too?